webui/react/.prettierrc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "printWidth": 100,
```

Comment:
You only need to include options that are different than the default here which would make it easier to see which settings are custom.

Reply:
Sometimes Prettier changes default such as https://prettier.io/docs/en/options.html#trailing-commas, so I wanna declarer some important ones explicitly

Reply:
These changes are only with major versions and our prettier version will be pinned. Better to simplify this here and then if we need to change it when upgrading prettier we will.

Reply:
The reason for the default there change was because of a broader change in trailing comma handling in the JS community